bigquery/bigquery_create_table_iam/main.tf

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START bigquery_create_table_iam]
+resource "google_bigquery_dataset" "default" {
+  dataset_id                      = "mydataset"
+  default_partition_expiration_ms = 2592000000  # 30 days
+  default_table_expiration_ms     = 31536000000 # 365 days
+  description                     = "dataset description"
+  location                        = "US"
+  max_time_travel_hours           = 96 # 4 days
+
+  labels = {
+    billing_group = "accounting",
+    pii           = "sensitive"
+  }
+}
+
+resource "google_bigquery_table" "default" {
+  dataset_id = google_bigquery_dataset.default.dataset_id
+  table_id   = "mytable"
+
+  schema = <<EOF
+[
+  {
+    "name": "ID",
+    "type": "INT64",
+    "mode": "NULLABLE",
+    "description": "Item ID"
+  },
+  {
+    "name": "Item",
+    "type": "STRING",
+    "mode": "NULLABLE"
+  }
+]
+EOF
+
+}
+
+data "google_iam_policy" "default" {
+  binding {
+    role = "roles/bigquery.dataOwner"
+    members = [
+      "user:raha@altostrat.com",
```

Comment:
Is this ok to use in public-facing content?

Reply:
Yes, it is one of the approved names in our style guide.